### ISSUE TITLE
Some way to completely ignore a column and leave it unchanged when sorting

### ISSUE THREAD

**Issue by harrison-munitz** [Mottie/tablesorter#1775]
I have a column that I'd like to use as static row numbers. When I sort the table I want that column to remain unchanged so that it stays in 1,2,3... order. Is this possible currently or else could it be added? Presumably some sort of 'sort-ignore' class or similar on the <th> tag?

**Comment by Mottie** [comment]
Hi @harrison-munitz!

Check out the two "automatic row numbering" widgets under "other" on the wiki home page - https://github.com/Mottie/tablesorter/wiki#widgets - to see if that accomplishes what you want.

**Comment by harrison-munitz** [comment]
Yes this does appear to do what I need. Although it still seems a bit odd that there's no way to exclude a column in general.
